src/Services/Search/ParametersHandler.php

```diff
@@ -71,14 +84,18 @@ public function handlePaginationAndSorting($search, $request)
      */
     public function getSortingOptions()
     {
-        return [
-            'item.score' => $this->getTranslator()->trans("Findologic::search.sortRelevance"),
-            'sorting.price.avg_asc' => $this->getTranslator()->trans("Findologic::search.sortPriceAsc"),
-            'sorting.price.avg_desc' => $this->getTranslator()->trans("Findologic::search.sortPriceDesc"),
-            'texts.name1_asc' => $this->getTranslator()->trans("Findologic::search.sortLabelAsc"),
-            'default.recommended_sorting' => $this->getTranslator()->trans("Findologic::search.sortSalesFrequency"),
-            'variation.createdAt_desc' => $this->getTranslator()->trans("Findologic::search.sortDateAdded")
-        ];
+        /** @var CeresConfig $config */
+        $config = $this->getConfig();
+
+        $returnArray = [];
+
+        foreach ($config->sorting->data as $data) {
+            if (in_array($data, Plugin::API_SORT_ORDER_AVAILABLE_OPTIONS)) {
+                $returnArray[$data] = $this->getTranslator()->trans("Findologic::Sorting.sortOption-" . $data);
```

Comment:
Translating this yourself shouldn't be necessary since Plentymarkets already takes care of that (assuming `$returnArray` has the right structure). Try
```
$returnArray[$data] = "Ceres::Template." . \Ceres\Helper\SearchOptions::TRANSLATION_MAP[$data]
```

If this change indeed works, you can safely delete `resources/lang/de/Sorting.properties` and `resources/lang/en/Sorting.properties`.

You can also check `\Ceres\Helper\SearchOptions` to see Plentymarkets default implementation.